router.go

```diff
@@ -17,11 +17,12 @@ var (
 	// ChannelSize is the buffer size used by so-called actor goroutines
 	// throughout mesh.
 	ChannelSize = 16
+
+	gossipInterval = 30 * time.Second
```

Comment:
Having this as a package-scoped variable will get confusing if two different callers try to configure different values.  I think it would need to be plumbed through from config, which looks fairly straightforward.

Reply:
Made gossip interval as router configuration property. PTAL